File: wlxx-camunda/wlxx-camunda-provider/src/main/java/com/xzg/wlxx/camunda/delegate/DemoDelegate.java
package com.xzg.wlxx.camunda.delegate;

import lombok.extern.slf4j.Slf4j;
import org.camunda.bpm.engine.delegate.DelegateExecution;
import org.camunda.bpm.engine.delegate.JavaDelegate;
import org.springframework.stereotype.Component;

@Component
@Slf4j
public class DemoDelegate implements JavaDelegate {
    @Override
    public void execute(DelegateExecution execution) throws Exception {
        log.info(execution.getId());
    }
}

File: wlxx-camunda/wlxx-camunda-provider/src/main/kotlin/com/xzg/wlxx/camunda/delegate/DemoDelegate.kt
package com.xzg.wlxx.camunda.delegate

import com.xzg.wlxx.common.log.WlxxLog.Companion.log
import org.camunda.bpm.engine.delegate.DelegateExecution
import org.camunda.bpm.engine.delegate.JavaDelegate
import org.springframework.stereotype.Component

@Component
class DemoDelegate : JavaDelegate {
    @Throws(Exception::class)
    override fun execute(execution: DelegateExecution) {
        log.info(execution.id)
    }
}
